ibis/expr/tests/test_datatypes.py

```diff
@@ -461,15 +461,6 @@ def test_time_valid():
                 'not supported'
             ),
         ),
-        param(
-            pd.Timedelta('3', unit='Y'),
```

Comment:
I think you don't need to remove the entire parameter .. but you can just replace `pd.Timedelta('3', unit='Y')` to `None`
probably it also test if `dt.Interval(unit='Y')` will raise an unsupported error ... but maybe I am wrong .. it would be good to check that before remove the entire parameter.

Reply:
Sounds good. Added this `param` back with `pd.Timedelta('3', unit='Y')` replaced by `None` and the `xfail` was successfully triggered